resources/user-agents/browsers/android-browser/android-browser-4-0.json

```diff
@@ -1276,8 +1287,7 @@
             "A5000": "Sony A5000"
           },
           "platforms": [
-            "Android_4_0",
-            "Android"
```

Comment:
I removed "Android" here because it was causing the Lite and Standard files to catch some UAs with `SM-A5000` as this device, which was incorrect.  The UA in the test suite for this pattern is an Android 4.0 UA, so I 'm not sure why the generic Android platform was added here.